### PR TITLE
feat(docs): add register services on deltav guide

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**

### CODE REVIEW

File: pages/guides/agentverse/registering-agent-services.mdx [hunk 1]
```diff
@@ -2,105 +2,120 @@ import { Callout } from 'nextra/components'
...
+    class Request(Model):
+        message: str
 
 
-@simples.on_message(model=Request, replies={UAgentResponse})
-async def handle_message(ctx: Context, sender: str, msg: Request):
-    await ctx.send(sender, UAgentResponse(message="0", type=UAgentResponseType.FINAL))
+    @simples.on_message(model=Request, replies={UAgentResponse})
+    async def handle_message(ctx: Context, sender: str, msg: Request):
+        await ctx.send(sender, UAgentResponse(message="0", type=UAgentResponseType.FINAL))
```
Comment (pratrivedi) [inline pages/guides/agentverse/registering-agent-services.mdx:22]: Suggestion : Where we are using `UagentResponseType` , we can explain the `ResponseTypes` to user
`UAgentResponseType`  is used to represent different response types of the agent. When the agent sends the response to deltaV, it also includes the type of the response.
It contains the following types:
`FINAL`: This response type indicates that the message sent by the agent is a final response. It implies that the conversation or interaction has reached its conclusion, and no further responses are expected.
`ERROR`: This response type is used when the agent encounters an error or an exceptional situation
`SELECT_FROM_OPTIONS`: This response type is used when the deltaV expects the user to select one option from a list of available options.
`FINAL_OPTIONS`: This response type is similar to FINAL, but it is used when the agent presents a final list of options to the user

Comment (FelixNicolaeBucsa) [reply pages/guides/agentverse/registering-agent-services.mdx:22]: Integrated.

File: pages/guides/agentverse/registering-agent-services.mdx [hunk 2]
```diff
@@ -2,105 +2,130 @@ import { Callout } from 'nextra/components'
...
+    <Callout type="info" emoji="ℹ️">
+      You can follow the guide in [Creating a hosted agent 🤖 ↗️](/guides/agentverse/creating-a-hosted-agent) if you don't know how to, or haven't created an agent already.
+    </Callout>
 
 This is a simple example of an agent, it does nothing but respond to any agent that sends it a message with `0`.
 
-<Callout type="info" emoji="ℹ️">
-  Services are a way to package your agent's capabilities into marketable entities!
-</Callout>
+The Agentverse Section shows its potential when operating on the [DeltaV ↗️](/concepts/deltav/intro) platform, given that AI Agents and related services registered within the Agentverse, are those retrieved by users when interacting with DeltaV AI Agent through the DeltaV chat interface to require any type of service they need.
```
Comment (devjsc) [inline pages/guides/agentverse/registering-agent-services.mdx]: Why do we have so much self-props/marketing copy in the docs? It always puts me off when I read this kind of sentence

File: pages/guides/agentverse/registering-agent-services.mdx [hunk 3]
```diff
@@ -2,105 +2,130 @@ import { Callout } from 'nextra/components'
...
-    - Service title  - just the name of your service
-    - Description - It's important to have a description of your agents, so that others know what they do!
-    - Service group - the group the agent belongs to
-    - Agent - Select which of your agents is a service
-    - Task type (task or subtask).
-    - Protocol - it's defined in your Agent.
-    - Model - again, we defined that in simple_protocol.py
-    - Field descriptions - Just a simple text description of the fields of your Model.
+    - **Service title**: just the name of your service.
+    - **Description**: It's important to have a description of your agents, so that others know what they do!
```
Comment (devjsc) [inline pages/guides/agentverse/registering-agent-services.mdx]:
```suggestion
    - **Description**: Super important to be as detailed as you can, as reasoning engine looks at descriptions to understand what your service does.
```

File: pages/guides/agentverse/registering-agent-services.mdx [hunk 4]
```diff
@@ -2,105 +2,130 @@ import { Callout } from 'nextra/components'
...
 
-## Let's find our service on DeltaV
+    @agent.on_interval(period=3.0)
+    async def send_message(ctx: Context):
+        await ctx.send('YOUR AGENT ADDRESS', Request(message="hello there bob"))
+        ctx.logger.info(f"Message has been sent to basically zero")
+    ````
 
-Head to [DeltaV ↗️](https://deltav.agentverse.ai/) and sign in.
+## Find your service on DeltaV
```
Comment (devjsc) [inline pages/guides/agentverse/registering-agent-services.mdx]:
```suggestion
## Let's find our service on DeltaV
```

feels more natural, it's a guide and personable, i don't want to lose character from the guides

File: pages/guides/agentverse/registering-agent-services.mdx [hunk 5]
```diff
@@ -2,105 +2,130 @@ import { Callout } from 'nextra/components'
...
 
 ![](../../../src/images/guides/agentverse/deltav_simple_start.png)
 
 Then, this is the full output:
 
 ![](../../../src/images/guides/agentverse/deltav_full_zero.png)
 
-With that, you've got a service which can be discovered and contacted with DeltaV. Awesome.
+    <Callout type="info" emoji="ℹ️">
+      Checkout our additional guide [Create and register a service on DeltaV 🌐 ↗️](/guides/deltav/create-and-get-a-service-on-deltav) for an additional resource on how to register your agent as a service and retrieve it within DeltaV.
```
Comment (devjsc) [inline pages/guides/agentverse/registering-agent-services.mdx]: what's the difference?

File: pages/guides/agentverse/registering-agent-services.mdx (hunk 2 above)
Comment (FelixNicolaeBucsa) [reply pages/guides/agentverse/registering-agent-services.mdx]: I thought i had removed that "Services are a way to package your agent's capabilities into marketable entities!" sentence. Am checking again and fix

File: pages/guides/agentverse/registering-agent-services.mdx (hunk 5 above)
Comment (FelixNicolaeBucsa) [reply pages/guides/agentverse/registering-agent-services.mdx]: It shows a real use case (EV Charger). Only that is the difference. Hadn't removed as i was waiting for your feedback before doing so.

**Pull request merged**